module_utils/.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
remove this file (at .vscode to .gitignore)

Reply:
okay, will remove it through .gitignore